initial_setup/__init__.py

```diff
@@ -113,11 +121,26 @@ def __init__(self, gui_mode):
         setup_ifcfg_log()
 
     @property
+    def external_reconfig(self):
```

Comment:
Well, I think it is still beneficial due to being able to attach a proper docstring to it.